README.md

```diff
@@ -8,92 +8,193 @@
 Metaprogramming tools for DataFrames.jl objects.
 These macros improve performance and provide more convenient syntax.
 
-# Features
+DataFrames.jl has the functions `select`, `transform`, and `combine` 
+for manipulating data frames. DataFramesMeta provides the macros 
+`@select`, `@transform`, and `@combine` to mirror these functions with 
+more convenient syntax. Inspired by [dplyr](https://dplyr.tidyverse.org/) in R 
+and [LINQ](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/linq/)
+in C#. 
```

Comment:
`...inspired by dplyr and LINQ` is crisper and "in R and In C# are just superfluous to me.